mail/verification_api.py

```diff
@@ -32,3 +32,31 @@ def send_verification_email(
             EMAIL_VERIFICATION,
         )
     )
+
+
+def send_verify_email_change_email(request, change_request):
```

Comment:
Missed this in my last review, can you add a test for this as well?

Reply:
done